apis/apps/v1alpha1/workloadspread_types.go

```diff
@@ -48,6 +53,12 @@ type TargetReference struct {
 	Name string `json:"name"`
 }
 
+// TargetFilter is used to filter the Pods managed by WorkloadSpread from the TargetReference.
+type TargetFilter struct {
+	Selector         *metav1.LabelSelector `json:"selector,omitempty"`
+	ReplicasPathList []string              `json:"replicasPathList,omitempty"`
```

Comment:
is these two field optional, if that is the case, consider add "+optional" markup